chromadb/test/property/invariants.py

```diff
@@ -39,18 +39,16 @@ def wrap_all(record_set: RecordSet) -> NormalizedRecordSet:
         embedding_list = None
     elif isinstance(record_set["embeddings"], list):
         assert record_set["embeddings"] is not None
-        if len(record_set["embeddings"]) > 0 and not all(
-            isinstance(embedding, list) for embedding in record_set["embeddings"]
-        ):
-            if all(isinstance(e, (int, float)) for e in record_set["embeddings"]):
+        if len(record_set["embeddings"]) > 0 and not all(isinstance(embedding, (list, np.ndarray)) for embedding in record_set["embeddings"]):
```

Comment:
Similar to the OneOrMany / Normalize conversion, lets explicitly split this up into the two types we are dealing with.